addons/binding/org.openhab.binding.sensebox/ESH-INF/thing/box.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sensebox" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- This thing represents the Arduino box that sends data to api.opensensebox.org . -->
+	<thing-type id="box">
+		<label>Sensebox</label>
+		<description>Sensor Box</description>
+
+		<channel-groups>
+			<channel-group typeId="descriptors" id="descriptors"/>
```

Comment:
> Neither can I bind them to Items to use in rules or sitemaps.

Well, if they are static, there really isn't much sense in doing so. And for showing static Thing properties on a Text widget in UIs, I would rather prefer to enhance the sitemaps to be able to reference a thing property for a text widget to display.

So maybe let's look at the channels/properties individually:
- location: Ok, if it can change, let's keep it as a channel.
- imageUrl: If you want to use images, I would actually suggest to use an ImageItem for it
- name + exposure: I would keep that as properties really
- apiUrl + mapUrl: Is this in any way relevant for UIs at all? And if, wouldn't it be good enough to show it on the Thing as properties of it?